docs/source/developer_instructions.rst

```diff
@@ -43,6 +43,24 @@ Run ``python -m pytest`` to run all unittests defined in the subfolder
 `pytest-runner <https://pypi.python.org/pypi/pytest-runner>`_.
 
 
+Snowflake testing
```

Comment:
Thank you for updating the developer docs -- we aren't always consistent with this so it's very helpful.